keras_cv/layers/preprocessing/random_zoom.py

```diff
@@ -28,7 +28,7 @@
 
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class RandomZoom(BaseImageAugmentationLayer):
+class RandomZoom(VectorizedBaseImageAugmentationLayer):
```

Comment:
can we add a unit test to ensure that each image is zoomed independently?

This can be achieved by:

- generate an image using random.normal((1, 100, 100, 3))
- generate a batch with tf.stack([image, image], axis=0)
- call a random zoom layer
- assertNotAllClose(result[0], result[1])

This ensures we still have element-wise augmentation

Reply:
Done.

I added a unit test:
```python
    def test_random_zoom_on_batched_images_independently(self):
        image = tf.random.uniform(shape=(100, 100, 3))
        input_images = tf.stack([image, image], axis=0)

        layer = RandomZoom(
            height_factor=(-0.4, -0.5), width_factor=(-0.2, -0.3)
        )

        results = layer(input_images)
        self.assertNotAllClose(results[0], results[1])
```
and it passed as expected.